users/ignoring.rst

```diff
@@ -115,9 +115,10 @@ The ``.stignore`` file contains a list of file or path patterns. The
 Example
 -------
 
-Given a directory layout::
+Given a synced folder directory layout::
```

Comment:
This isn't correct, what's displayed is a "directory layout", it's the filesystem objects not any Syncthing concept. If you want to be explicit about the folder root part, that could be the following (though that feels overly verbous):

```suggestion
Given a directory layout starting at the synced folder's root::
```